src/categorical_algebra/Limits.jl

```diff
@@ -40,7 +40,7 @@ abstract type AbstractLimit{Ob,Diagram} end
 
 ob(lim::AbstractLimit) = apex(lim)
 cone(lim::AbstractLimit) = lim.cone
-apex(lim::AbstractLimit) = apex(cone(lim))
+apex(lim::AbstractLimit) = apex(cone(lim)) #isn't this a synonym for ob(lim)?
```

Comment:
Ah, OK. I feel like since Multispans are concrete it'd be a bit more aesthetic to define something like `multispan(X::Limit)=X.cone` and `apex(X::Limit)=apex(multispan(X))`, but it doesn't seem worth bothering.